static/js/tools/timeline_info.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+
+class Info extends Component {
+    shouldComponentUpdate = () => false
+    render() {
+        return (
+            <div id="placeholder-container">
+                <p> Welcome to Data Commons. This tool is to help you
+                    find data about places (zip codes, cities, counties, states) from a variety of
+                    sources, including the Census, FBI, Bureau of Labor Statistics, CDC and others. </p>
+                <p> Enter a place in the search box above and then pick one or more of the variables in
+                    the pane to the left. There are thousands of variables to choose from, arranged in a
+                    topical hierarchy. </p>
+                <p> Or you can start your exploration from these interesting points ...</p>
+                <ul>
+                    <li>
+                        <b>University towns</b> by
+                        <a href="#&place=geoId/0606000,geoId/2511000,geoId/2603000,geoId/1777005,geoId/1225175,geoId/4815976&statsvar=MedianAge'Demographics'Median%20age"> age</a>,
```

Comment:
please also change here. As this is used a lot, can think of a short name for it (2 or 3 letters)

Reply:
done